### DISCUSSION TITLE
Missing tslib/modules/index.js after "nuxi build" in .output/server/node_modules

### DISCUSSION THREAD

**Discussion by josefryzi** [nuxt/nuxt#21533] [Questions] [answered]
Hello everyone, 
really interesting issue on my side.

I am using lates nuxt@3.5.3 with this package.json

```
{
  "private": true,
  "scripts": {
    "build": "nuxi build",
    "dev": "nuxi dev",
    "start": "node .output/server/index.mjs"
  },
  "overrides": {
    "vue": "latest"
  },
  "engines": {
    "node": "16.x.x"
  },
  "devDependencies": {
    "@nuxt/devtools": "^0.5.5",
    "@nuxtjs/apollo": "^5.0.0-alpha.6",
    "@nuxtjs/tailwindcss": "^6.6.4",
    "autoprefixer": "^10.4.14",
    "nuxt": "^3.5.3",
    "postcss": "^8.4.21",
    "sass": "^1.59.3",
    "tailwindcss": "^3.2.7"
  },
  "dependencies": {
    "@nuxt/image": "^1.0.0-rc.1",
    "@pinia/nuxt": "^0.4.11",
    "@vuelidate/core": "^2.0.2",
    "@vuelidate/validators": "^2.0.2",
    "click-outside-vue3": "^4.0.1",
    "consola": "^3.1.0",
    "date-fns": "^2.29.3",
    "jwt-decode": "^3.1.2",
    "pinia": "^2.0.33",
    "vue-toastification": "^2.0.0-rc.5",
    "vue3-touch-events": "^4.1.3"
  }
}
```

But after I run "nuxi build" and trying to see preview with "npm run start" I am having issue in console: 

```
[nuxt] [request error] [unhandled] [500] Cannot find module '/Users/josefryzi/projects/sve-fe/.output/server/node_modules/tslib/modules/index.js' imported from /Users/josefryzi/projects/sve-fe/.output/server/chunks/app/server.mjs
Did you mean to import tslib/tslib.js?
```

Any ideas how to solve this issue? Thank you!

**Comment by albertotesoro** [comment]
same issue here :-(

{
  "name": "xxxxx",
  "version": "1.0.16",
  "license": "none",
  "private": true,
  "engines": {
    "node": ">=18 <19"
  },
  "scripts": {
    "dev": "nuxt dev",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare && npm rebuild --platform=linux --arch=x64 --target=18x sharp && npm rebuild --platform=linux --arch=arm64 --target=18x sharp",
    "Zapatos": "npx zapatos",
    "GENERATE-GQL-TYPES-CLIENT": "graphql-code-generator --config codegen.yml",
    "YARN-UPDATE": "node ./scripts/yarn-update",
    "release-it": "yarn version --patch && git push --follow-tags"
  },
  "devDependencies": {
    "@babel/core": "^7.21.8",
    "@babel/eslint-parser": "^7.21.3",
    "@ckeditor/ckeditor5-build-classic": "^38.0.1",
    "@ckeditor/ckeditor5-vue": "^5.0.0",
    "@ckeditor/vite-plugin-ckeditor5": "^0.1.2",
    "@graphql-codegen/add": "^5.0.0",
    "@graphql-codegen/cli": "^4.0.1",
    "@graphql-codegen/fragment-matcher": "^5.0.0",
    "@graphql-codegen/graphql-modules-preset": "^4.0.0",
    "@graphql-codegen/introspection": "^4.0.0",
    "@graphql-codegen/near-operation-file-preset": "^2.5.0",
    "@graphql-codegen/typed-document-node": "^5.0.0",
    "@graphql-codegen/typescript": "^4.0.0",
    "@graphql-codegen/typescript-apollo-client-helpers": "^2.2.6",
    "@graphql-codegen/typescript-operations": "^4.0.0",
    "@graphql-codegen/typescript-resolvers": "^4.0.0",
    "@graphql-codegen/typescript-vue-apollo": "^3.3.7",
    "@nuxt/content": "^2.7.0",
    "@nuxt/devtools": "^0.6.1",
    "@nuxtjs/apollo": "^5.0.0-alpha.5",
    "@nuxtjs/eslint-config-typescript": "^12.0.0",
    "@nuxtjs/google-fonts": "^3.0.1",
    "@nuxtjs/html-validator": "^1.2.3",
    "@nuxtjs/tailwindcss": "^6.7.2",
    "@serverless-stack/resources": "^1.18.4",
    "@sidebase/nuxt-auth": "^0.6.0-beta.0",
    "@types/apollo-upload-client": "^17.0.2",
    "@types/ckeditor__ckeditor5-editor-classic": "^37.0.0",
    "@types/ckeditor__ckeditor5-heading": "^37.0.0",
    "@types/cors": "^2.8.13",
    "@types/formidable": "^2.0.6",
    "@types/fs-extra": "^11.0.1",
    "@types/jsonwebtoken": "^9.0.2",
    "@types/lodash.clonedeep": "^4.5.7",
    "@types/lodash.orderby": "^4.6.7",
    "@types/mime-types": "^2.1.1",
    "@types/node": "^18.16.3",
    "@types/pg": "^8.6.6",
    "@types/uuid": "^9.0.1",
    "@typescript-eslint/parser": "^5.59.8",
    "@vite-pwa/nuxt": "^0.1.0",
    "@vueuse/components": "^10.2.0",
    "@vueuse/core": "^10.2.0",
    "@vueuse/nuxt": "^10.2.0",
    "aws-cdk-lib": "^2.84.0",
    "dotenv": "^16.3.0",
    "eslint": "^8.43.0",
    "eslint-config-prettier": "^8.8.0",
    "eslint-plugin-nuxt": "^4.0.0",
    "eslint-plugin-prettier": "^4.2.1",
    "fs-extra": "^11.1.1",
    "graphql-codegen": "^0.4.0",
    "nuxi": "^3.5.3",
    "nuxt": "^3.5.3",
    "nuxt-delay-hydration": "^1.1.4",
    "nuxt-icon": "^0.4.0",
    "nuxt-security": "^0.13.0",
    "nuxt-typed-router": "^3.2.2",
    "postcss-nesting": "^11.3.0",
    "prettier": "^2.8.8",
    "sass": "^1.63.4",
    "serverless-domain-manager": "^7.0.4",
    "serverless-dotenv-plugin": "^6.0.0",
    "serverless-offline": "^12.0.4",
    "ts-node": "^10.9.1",
    "vue-tsc": "^1.8.0",
    "zapatos": "^6.1.4"
  },
  "dependencies": {
    "@apollo/server": "^4.7.4",
    "@as-integrations/h3": "^1.1.5",
    "@aws-sdk/client-s3": "^3.354.0",
    "@ckeditor/ckeditor5-basic-styles": "^38.0.1",
    "@ckeditor/ckeditor5-dev-utils": "^38.0.1",
    "@ckeditor/ckeditor5-editor-classic": "^38.0.1",
    "@ckeditor/ckeditor5-essentials": "^38.0.1",
    "@ckeditor/ckeditor5-html-support": "^38.0.1",
    "@ckeditor/ckeditor5-link": "^38.0.1",
    "@ckeditor/ckeditor5-paragraph": "^38.0.1",
    "@ckeditor/ckeditor5-source-editing": "^38.0.1",
    "@ckeditor/ckeditor5-theme-lark": "^38.0.1",
    "@ckeditor/ckeditor5-upload": "^38.0.1",
    "@editorjs/editorjs": "^2.27.0",
    "@editorjs/header": "^2.7.0",
    "@mdi/font": "^7.1.96",
    "@nuxt/image-edge": "^1.0.0-rc.1-28115526.1a8c782",
    "@nuxt/kit-edge": "^3.6.0-28115499.c73bed8c",
    "@nuxtjs/i18n": "^8.0.0-beta.10",
    "@nuxtjs/robots": "^3.0.0",
    "@pinia/nuxt": "^0.4.10",
    "@stylify/nuxt": "^0.6.1",
    "@types/lodash.merge": "^4.6.7",
    "@vueup/vue-quill": "^1.1.1",
    "apollo-upload-client": "^17.0.0",
    "camelcase-keys": "^8.0.2",
    "cors": "^2.8.5",
    "critical": "^5.1.1",
    "dataloader": "^2.2.2",
    "dayjs": "^1.11.7",
    "defu": "^6.1.2",
    "exifr": "^7.1.3",
    "formidable": "^3.3.2",
    "graphql": "^16.6.0",
    "graphql-scalars": "^1.20.1",
    "graphql-tag": "^2.12.6",
    "graphql-upload-ts": "^2.0.6",
    "h3-formidable": "^0.3.2",
    "jsonwebtoken": "^9.0.0",
    "lodash.clonedeep": "^4.5.0",
    "lodash.merge": "^4.6.2",
    "lodash.orderby": "^4.6.0",
    "lorem-ipsum": "^2.0.8",
    "mime-types": "^2.1.35",
    "next-auth": "4.22.1",
    "nuxt-graphql-client": "^0.2.25",
    "nuxt-icons": "^3.2.1",
    "nuxt-og-image": "^1.5.9",
    "pathe": "^1.1.1",
    "pg": "^8.9.0",
    "serverless-apigw-binary": "^0.4.4",
    "serverless-http": "^3.2.0",
    "snake-case": "^3.0.4",
    "string-strip-html": "^13.2.3",
    "typescript": "^5.1.3",
    "ufo": "^1.1.2",
    "uuid": "^9.0.0",
    "vite-plugin-vuetify": "^1.0.2",
    "vue-file-agent": "^1.7.3",
    "vue-filepond": "^7.0.3",
    "vue3-carousel": "^0.3.1",
    "vuedraggable": "^4.1.0",
    "vuetify": "^3.2.4",
    "yarn": "^1.22.19"
  },
  "lint-staged": {
    "*.{ts,vue}": [
      "eslint --fix"
    ]
  }
}

**Comment by albertotesoro** [comment, marked as answer]
just solved adding  "tslib" to build.transpile in nuxt.config.ts

**Reply by nyambe** [reply]
This fixed it, but what is the problem.  Is this a bug? should we roll back from 3.5?

**Reply by albertotesoro** [reply]
i don't know... i have a rule: whenever receive a non-sense error, don't think, first put something in transpile ^_^

**Reply by albertotesoro** [reply]
just upgraded to 3.6.1: issue is still there

**Reply by SGB555** [reply]
just upgraded to 3.6.3: issue is still there

**Reply by SebbeJohansson** [reply]
How is this still an issue?

**Reply by cinco-de-mauro** [reply]
On 3.7.1 and ran into this. Thanks!

**Comment by hazadus** [comment]
> just solved adding "tslib" to build.transpile in nuxt.config.ts

Thanks, this helped me, too!

**Comment by cmstolper** [comment]
This may be related to the use of [tslib](https://github.com/microsoft/tslib) via @nuxtjs/apollo. See:
- https://github.com/nuxt/nuxt/issues/19265
- https://github.com/vueuse/motion/issues/102
- https://github.com/nuxt-modules/apollo/issues/516
